content/blog/2024-04-04-tmix-03.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "This Month in Xilem, March 2024"
+authors = ["Daniel McNab"]
++++
+
+<!-- I don't think we need to set the date, as it is inferred from the slug -->
```

Comment:
Specifically, this is mostly just trying to get this comment thread, to see if we can get an explanation as to why the full date/time is set in the other posts.

I don't think it's useful to have in the final version, i.e.
```suggestion
+++

```

Reply:
I assume it's in the other posts because this feature was not known. Can see what Raph has to say about it I guess.

There's a tiny bit of value in having that comment stay so that future post writes can more easily figure out why the date suddenly doesn't appear. Not a big issue though as the set of writers here is very limited.